documentation/src/main/asciidoc/getting_started/installation.adoc

```diff
@@ -11,27 +11,32 @@ TIP: If you already have any of these pieces of software, there is no need to in
 === JDK
 Choose your Java runtime, and follow their installation instructions. For example, you could choose one of:
 
-*  link:http://openjdk.java.net/install/[OpenJDK] 
+*  link:http://openjdk.java.net/install/[OpenJDK]
 *  link:http://www.java.com/en/download/manual.jsp[Oracle Java SE]
-*  link:http://download.oracle.com/docs/cd/E15289_01/doc.40/e15065/toc.htm[Oracle JRockit] 
+*  link:http://download.oracle.com/docs/cd/E15289_01/doc.40/e15065/toc.htm[Oracle JRockit]
```

Comment:
Off-topic, but maybe we should remove references to JRockit from the docs.

Reply:
Good call. Thanks @danberindei I've removed those references and will update the PR.